.github/workflows/publish.yaml

```diff
@@ -28,10 +28,52 @@ jobs:
           node-version: 18.12.1
 
       - name: Build website
-        run: npm install gatsby-cli && npm run build
+        run: npm install gatsby-cli && npm run clean &&npm run build
 
-      - name: Sync files to S3
-        run: aws s3 sync ./public/ s3://$AWS_BUCKET_NAME --acl public-read --delete --cache-control max-age=604800
```

Comment:
`--cache-control max-age=604800` means 7 days. This might be the reason why we get the "black screen". An assumption is that when the website gets updated, old cached data do not correspond to the latest files, leading to an unexpected error.